drivers/display/Kconfig.smartbond

```diff
@@ -0,0 +1,18 @@
+# Smartbond display controller configuration options
+
+# Copyright (c) 2023 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config DISPLAY_SMARTBOND
+	bool "Smartbond display controller driver"
+	depends on DT_HAS_RENESAS_SMARTBOND_DISPLAY_ENABLED
+	select DMA
+	default y
+	help
+	  Enable Smartbond display controller.
+
+config DISPLAY_BUFFER_PSRAM
```

Comment:
This Kconfig should depend on `DISPLAY_SMARTBOND`, and should be namespaced- IE `DISPLAY_SMARTBOND_BUFFER_PSRAM`